sentences/de/_common.yaml

```
@@ -33,6 +33,12 @@ lists:
       type: "percentage"
       from: 0
       to: 100
+  brightness_level:
+    values:
+      - in: (max|(maximale|höchste|volle) (Helligkeit|Stufe)|maximum)
```

```suggestion
      - in: (max[(imale|imum)]|höchste|volle)
```
Sorry, but after some thought I'm think we should remove the `(Helligkeit|Stufe)` here and adjust the sentence definitions accordingly to create meaningful sentences. the same counts for the minimum values below.